packages/components/src/interfaces/native/Table.ts

```diff
@@ -42,7 +42,7 @@ const TableCellSchema = Type.Object({
 const TableHeaderCellSchema = Type.Object({
   type: Type.Literal("tableHeader"),
   attrs: Type.Optional(TableBaseCellSchema),
-  content: Type.Array(ParagraphSchema, {
+  content: Type.Array(Type.Ref(ParagraphSchema), {
```

Comment:
do we need more of this? i recall that our native components all use the `$ref` way

Reply:
Hmm for now not really, `$ref` was used quite a lot previously because we were manually writing the schema but now that it is auto-generated, the `$ref` is only used to break up circular/deep references.